content/cloud/custom-domains-tutorial.md

```diff
@@ -0,0 +1,73 @@
+
+## Select Your Spin Application
+
+In the [Fermyon Cloud Dashboard](https://cloud.fermyon.com), navigate to your Spin application of choice and click the "Manage Domains" button.
+
+![DRAFT SCREENSHOT - app panel view](/static/image/custom-domains/app-panel-view.png)
+
+## Input Your Custom Domain
+
+Select the "Add custom domain" button and input your desired domain name. 
+
+![DRAFT SCREENSHOT - step 1](/static/image/custom-domains/Step-1.png)
+
+Please add the complete domain name (i.e., `www.example.com`) as Fermyon Cloud does not support redirects at this time. Click save to confirm you would like to add this domain name to Fermyon Cloud.
+
+![DRAFT SCREENSHOT - step 2](/static/image/custom-domains/Step-2.png)
+
+Up next, you'll need to configure your DNS records with your registrar to point at Fermyon's name servers (NS). 
+
+>>If you click "I'll do this later" you will be pausing the workflow and your *custom domain will not work yet* with your Spin application. You can navigate back to the "Manage Domains" workflow to resume your journey. 
+
+![DRAFT SCREENSHOT - step 3](/static/image/custom-domains/Step-3.png)
+
+## Update Your Domain Registrar With Fermyon Cloud Name Servers
+
+At this step, Fermyon Cloud will check to see if you've previously associated your domain with Fermyon Cloud before. If this is the first time Fermyon Cloud has seen the domain, you will need to visit your domain registrar to delegate your domain to Fermyon Cloud. Domain delegation is the process of reassigning nameservers for a domain name to another DNS provider, which allows said nameservers to handle all DNS requests and queries for that domain. 
+
+In this step, you will update your existing nameserver records at your domain registrar to be replaced by Fermyon Cloud's namesever records. Fermyon Cloud will automatically generate four nameserver records for you. 
+
+![DRAFT SCREENSHOT - step 4](/static/image/custom-domains/Step-4.png)
+
+Please copy all four nameserver records and use them to replace your name server records with your domain registrar. Each domain registrar has its own unique set of instructions for completing this workflow. Please visit your domain registrar's documentation for instructions on how to update your nameserver records. For your convenience, we've sourced instructions for [Google Domains](https://support.google.com/domains/answer/3290309?hl=en#) and [Namecheap](https://www.namecheap.com/support/knowledgebase/article.aspx/767/10/how-to-change-dns-for-a-domain/).
+
+>> Note that you do not need to provide a SSL certificate. Fermyon Cloud creates one internally using Let's Encrypt, and will keep renewing it for the lifetime of your Spin application. 
+
+## Verify Domain Has Been Delegated
+
+Please wait for up to 72 hours for the process change. Once the verification process has been completed, you should see a green checkmark next to your custom domain. 
```

Comment:
Not sure if "process change" is a typo... "for the change to be processed"? "to process the change"?

Reply:
I like "for the change to be processed".